.eslintrc

```diff
@@ -0,0 +1,98 @@
+{
+  "root": true,
+  "extends": [
+    "eslint:recommended"
+  ],
+  "env": {
+    "es6": true,
+    "node": true
+  },
+  "parserOptions": {
+    "ecmaVersion": 2019,
+    "sourceType": "module"
+  },
+  "globals": {
+    "Cypress": false,
+    "cy": false,
+    "before": false,
+    "beforeEach": false,
+    "expect": false
+  },
```

Comment:
Why we're setting false to every global?

Reply:
`false` is equivalent for 'readonly'. I've pushed commit to change it to avoid misunderstanding